frontend/src/main/web/lib/components/DropDown.jsx

```diff
@@ -34,7 +34,7 @@ var DropDown = React.createClass({
       var buttonClassName = 'button--link txt--nowrap';
       buttonClassName += option === selected ? ' is-active' : '';
       return <li key={option} className='Dropdown-item'>
-        <button className={buttonClassName} onClick={self.handleOptionClick.bind(self, option)}>{option}</button>
+        <button className={buttonClassName} onClick={self.handleOptionClick.bind(self, option, onClickFn)}>{option}</button>
```

Comment:
Long line (and another nearby in this file). I like to put each attribute on a separate line when there are more than a couple or they are long.